meta/and.rb

```diff
@@ -1,4 +1,4 @@
-Mutant::Meta::Example.add do
+Mutant::Meta::Example.add :and do
```

Comment:
I would support that. Could also restrict the rule to just ruby DSL like `attr_reader`